<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcDialog</class>
 <widget class="QDialog" name="CalcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalcDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #99FF99;
color: #000;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnPercent">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>%</string>
         </property>
         <property name="shortcut">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnDivision">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="shortcut">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnMultiply">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>*</string>
         </property>
         <property name="shortcut">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnMinus">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="shortcut">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="2">
        <widget class="QPushButton" name="btnPlus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="shortcut">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" rowspan="2">
        <widget class="QPushButton" name="btnEqual">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="shortcut">
          <string>Enter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnPoint">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnC">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAC">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBracketOpen">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>(</string>
         </property>
         <property name="shortcut">
          <string>(</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBracketClose">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>)</string>
         </property>
         <property name="shortcut">
          <string>)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlusMinus">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
         <property name="shortcut">
          <string>\</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
